src/compiler/diagnosticMessages.json

```diff
@@ -3386,6 +3386,22 @@
         "category": "Error",
         "code": 4110
     },
+    "Class member cannot have override modifier because class '{0}' does not extended another class.": {
+        "category": "Error",
+        "code": 4111
+    },
+    "Class member cannot have override modifier because it's not existed in the base class '{0}'.": {
+        "category": "Error",
+        "code": 4112
+    },
+    "Class member must have override modifier because it's override the base class '{0}'.": {
+        "category": "Error",
+        "code": 4113
+    },
+    "'Override' modifier must be used with 'pedanticOverride' compiler option": {

```

Comment:
It should not be an error to use the `override` keyword outside of `pedanticOverride` or whatever the option gets called.